lib/common_test/src/ct_ssh.erl

```diff
@@ -494,438 +582,502 @@ subsystem(SSH, ChannelId, Subsystem, Timeout) ->
 %%%------------------------ SFTP COMMANDS --------------------------
 
 %%%-----------------------------------------------------------------
-%%% @spec sftp_connect(SSH) -> {ok,Server} | {error,Reason}
-%%%      SSH = connection()
-%%%      Server = pid()
-%%%      Reason = term()
 %%% @doc Starts an SFTP session on an already existing SSH connection.
 %%%      <code>Server</code> identifies the new session and must be
 %%%      specified whenever SFTP requests are to be sent.
+%%% @end
+-spec sftp_connect(SSH) -> {ok,Server} | {error,Reason} when
+      SSH :: connection(),
+      Server :: pid(),
+      Reason :: term().
 sftp_connect(SSH) ->
     call(SSH, sftp_connect).
 
 %%%-----------------------------------------------------------------
-%%% @spec read_file(SSH, File) -> Result
-%%%      SSH = connection()
-%%%      Result = ssh_sftp_return() | {error,Reason}
-%%%      Reason = term()
 %%% @doc For info and other types, see ssh_sftp(3).
+-spec read_file(SSH, File) -> ssh_sftp_return() | {error,Reason} when
+      SSH :: connection(),
+      File :: string(),
+      Reason :: term().
 read_file(SSH, File) ->
     call(SSH, {read_file,sftp,File}).
 %%%-----------------------------------------------------------------
-%%% @spec read_file(SSH, Server, File) -> Result
-%%%      SSH = connection()
-%%%      Result = ssh_sftp_return() | {error,Reason}
-%%%      Reason = term()
 %%% @doc For info and other types, see ssh_sftp(3).
+-spec read_file(SSH, Server, File) -> ssh_sftp_return() | {error,Reason} when
+      SSH :: connection(),
+      Server :: atom(),
```

Comment:
Should this really be an atom(). It is used as first argument to ssh_sftp:read_file, which is specified as ChannelPid::pid(). Isn't it the same as the result of sftp_connect/1?